flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/operator/KeyedCEPPatternOperator.java

```diff
@@ -40,25 +40,27 @@
 public class KeyedCEPPatternOperator<IN, KEY> extends AbstractKeyedCEPPatternOperator<IN, KEY, Map<String, IN>> {
 	private static final long serialVersionUID = 5328573789532074581L;
 
-	public KeyedCEPPatternOperator(TypeSerializer<IN> inputSerializer, boolean isProcessingTime, KeySelector<IN, KEY> keySelector, TypeSerializer<KEY> keySerializer, NFACompiler.NFAFactory<IN> nfaFactory) {
+	public KeyedCEPPatternOperator(
+		TypeSerializer<IN> inputSerializer,
+		boolean isProcessingTime,
+		KeySelector<IN, KEY> keySelector,
+		TypeSerializer<KEY> keySerializer,
+		NFACompiler.NFAFactory<IN> nfaFactory) {
```

Comment:
double indentation for method declarations is better because then one can more easily distinguish the parameter list from the body.